{"stats": {"errors": 0, "failures": 1, "passes": 10, "skipped": 0, "encoding": "UTF-8", "total": 11}, "results": [{"classname": "index_test:TestError", "name": "test_error", "time": 0.00020170211791992188, "type": "success"}, {"classname": "index_test:TestError", "name": "test_error_line_trace", "time": 9.560585021972656e-05, "type": "success"}, {"classname": "index_test:TestError", "name": "test_error_line_traces", "time": 0.0007364749908447266, "type": "failure", "errtype": "builtins.AssertionError", "message": "'error at 120' != 'error at 30.0'\n- error at 120\n?          ^^\n+ error at 30.0\n?          ^^^\n", "tb": "Traceback (most recent call last):\n  File \"/opt/conda/lib/python3.6/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/conda/lib/python3.6/unittest/case.py\", line 605, in run\n    testMethod()\n  File \"/home/immanence/evaluating-regression-lines-lab-data-science-intro-000/test/index_test.py\", line 40, in test_error_line_traces\n    self.assertEqual(traces[-1]['name'], 'error at 30.0')\n  File \"/opt/conda/lib/python3.6/unittest/case.py\", line 829, in assertEqual\n    assertion_func(first, second, msg=msg)\n  File \"/opt/conda/lib/python3.6/unittest/case.py\", line 1203, in assertMultiLineEqual\n    self.fail(self._formatMessage(msg, standardMsg))\n  File \"/opt/conda/lib/python3.6/unittest/case.py\", line 670, in fail\n    raise self.failureException(msg)\nAssertionError: 'error at 120' != 'error at 30.0'\n- error at 120\n?          ^^\n+ error at 30.0\n?          ^^^\n\n"}, {"classname": "index_test:TestError", "name": "test_regression_formula", "time": 0.00010991096496582031, "type": "success"}, {"classname": "index_test:TestError", "name": "test_rss", "time": 0.0003383159637451172, "type": "success"}, {"classname": "index_test:TestError", "name": "test_squared_error", "time": 9.441375732421875e-05, "type": "success"}, {"classname": "index_test:TestError", "name": "test_titles", "time": 9.608268737792969e-05, "type": "success"}, {"classname": "index_test:TestError", "name": "test_trace_rsme_value", "time": 0.0007302761077880859, "type": "success"}, {"classname": "index_test:TestError", "name": "test_x_values", "time": 8.296966552734375e-05, "type": "success"}, {"classname": "index_test:TestError", "name": "test_y_actual", "time": 0.00010156631469726562, "type": "success"}, {"classname": "index_test:TestError", "name": "test_y_values", "time": 8.7738037109375e-05, "type": "success"}]}